cpp/src/parquet/stream_writer.cc

```diff
@@ -208,6 +214,65 @@ void StreamWriter::CheckColumn(Type::type physical_type,
   }
 }
 
+int64_t StreamWriter::SkipColumns(int num_columns_to_skip) {
+  int num_columns_skipped = 0;
+
+  for (; (num_columns_to_skip > num_columns_skipped) &&
+         static_cast<std::size_t>(column_index_) < nodes_.size();
+       ++num_columns_skipped) {
+    const auto& node = nodes_[column_index_];
+
+    if (node->is_required()) {
+      throw ParquetException("Cannot skip column '" + node->name() +
+                             "' as it is required.");
+    }
+    auto writer = row_group_writer_->column(column_index_++);
+
+    switch (writer->type()) {
+      case Type::BOOLEAN:
+        static_cast<BoolWriter*>(writer)->WriteBatch(1, &def_level_zero_,
```

Comment:
please make "1" a constant with a meaningful name or add inline documentation for what 1 represents.

Reply:
I added a symbolic constant: kBatchSizeOne